Obsidian/Utils/WADHashGenerator.cs

```diff
@@ -177,13 +177,13 @@ private static IEnumerable<string> ProcessBINLinkedFiles(List<string> linkedFile
             foreach (string fetchedString in linkedFiles)
             {
                 strings.Add(fetchedString);
-                if (fetchedString.Contains("_skins_skin"))
+                if (fetchedString.Contains("_Skins_Skin"))
                 {
                     strings.AddRange(ProcessBINPackedLinkedFile(fetchedString));
                 }
             }
 
-            return strings.AsEnumerable();
+            return strings;
```

Comment:
Why did you remove `strings.AsEnumerable();` here but not anywhere else

Reply:
Uh just saw it while I was on that function. I can remove it and bundle this and a lot of other such changes in another PR if you prefer.

Reply:
yeah sure